payment-methods.adoc

```diff
@@ -0,0 +1,285 @@
+
+We welcome suggestions for new payment methods on Bisq, especially for those that enable new markets. Documentation on the criteria and process for adding new payment methods https://github.com/bisq-network/bisq-docs/issues/172[is forthcoming^]. For now, feel free to make suggestions by reaching out on https://bisq.network/slack-invite[Slack^] or https://keybase.io/team/bisq[Keybase^].
+
+Below is a list of fiat payment methods Bisq currently supports.
+
+CAUTION: The maximum trade sizes listed below are not available for most newly-created payment accounts. Please see the <<#account-signing, Account Signing>> section below for details on how to enable bigger trade sizes for your payment accounts.
+
+|===
+|Payment Method |Region |Trading Period |Per-Trade Limit|Notes
+
+|Advanced Cash
+|Global
+|1 day
+|{very-low-risk}
+|Not available in the USA
+
+|Alipay
+|China
+|1 day
+|{low-risk}
+|
+
+|Cash Deposit
+|N/A
+|4 days
+|{high-risk}
+|Some banks don't accept cash deposits from non-customers
+
+|Chase QuickPay
+|USA
+|1 day
+|{high-risk}
+|
+
+|Face-to-Face (F2F)
+|Global
+|4 days
+|{low-risk}
+|
+
+|Faster Payments
+|UK
+|1 day
+|{high-risk}
+|
+
+|HalCash
+|Spain
+|1 day
+|{low-risk}
+|
+
+|Interac e-Transfer
+|Canada
+|1 day
+|{high-risk}
+|
+
+|Japan Zengin Furikomi
+|Japan
+|1 day
+|{low-risk}
+|
+
+|MoneyBeam (N26)
+|Europe
+|1 day
+|{high-risk}
+|
+
+|MoneyGram
+|Global
+|4 days
+|{mid-risk}
+|
+
+|National Bank Transfer
+|N/A
+|4 days
+|{high-risk}
+|
+
+|Perfect Money
+|Europe and USA
+|1 day
+|{low-risk}
+|
+
+|Popmoney
+|USA
+|1 day
+|{high-risk}
+|
+
+|PromptPay
+|Thailand
+|1 day
+|{low-risk}
+|
+
+|Revolut
+|Global
+|1 day
+|{high-risk}
+|
+
+|SEPA
+|Europe
+|6 days
+|{high-risk}
+|
+
+|SEPA Instant
+|Europe
+|1 day
+|{high-risk}
+|
+
+|Swish
+|Sweden
+|1 day
+|{low-risk}
+|
+
+|Transfer with Same Bank
+|N/A
+|2 days
+|{high-risk}
+|
+
+|Transfer with Specific Banks
+|N/A
+|4 days
+|{high-risk}
+|
+
+|US Postal Money Order
+|USA
+|8 days
+|{high-risk}
+|Maximum is $1,000 per money order within the USA
+
+|Uphold
+|Global
+|1 day
+|{high-risk}
+|
+
+|WeChat Pay
+|China
+|1 day
+|{low-risk}
+|
+
+|Western Union
+|Global
+|4 days
+|{mid-risk}
+|
+
+|Zelle
+|USA
+|4 days
+|{high-risk}
+|
+
+|===
+
+== Account Signing
+
+Payment account _aging_ is the primary mechanism that determines how much bitcoin can be bought or sold with a particular payment account. For some types of payment accounts, account _signing_ determines when that aging begins.
+
+For lower-risk payment accounts, aging begins right away, as soon as the payment account is created in Bisq—no signing is required.
+
+For higher-risk payment accounts, aging begins only when the payment account is signed by a trusted peer. **Until 30 days _after_ this signing, maximum trade size is 0.01 BTC**.
```

Comment:
Only the buy size is 0.01 BTC, the sell size is 0.25 x max trade size for the payment type (which means it's 0.0625 BTC atm).